docs/reference/docker_compose_restart.yaml

```diff
@@ -11,7 +11,7 @@ long: |-
     If you are looking to configure a service's restart policy, please refer to
     [restart](https://github.com/compose-spec/compose-spec/blob/master/spec.md#restart)
     or [restart_policy](https://github.com/compose-spec/compose-spec/blob/master/deploy.md#restart_policy).
-usage: docker compose restart
+usage: docker compose restart [SERVICE...]
```

Comment:
This change should be done [here](https://github.com/docker/compose/blob/def189fae1a74a9cac327781aabeaa214d6906a9/cmd/compose/restart.go#L38)